samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -87,40 +99,52 @@ Testing
-   Press the **Mesh Light Switch** node in the list.
-   You'll see basic information about your mesh node and its configuration.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned mesh devices.
+#. Select the :guilabel:`Mesh Light Switch` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select the OOB method and follow the instructions in the app.
+
+Once the provisioning and initial configuration is complete, the app will go back to the Network screen.
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Light Switch` node.
+   Basic information about the mesh node and its configuration is displayed.
 #. In the Mesh node view, expand the first element.
    It contains the list of models in the first element of the node.
-#. Press the "Generic OnOff Client" model to see its configuration.
-#. Models must be bound to application keys to be open to communication.
-   Do this by pressing "BIND KEY" at the top of the screen.
-   Select "Application Key 1" from the list.
-#. In addition to the application key binding, Client models need publish parameters to be configured.
-   The publish parameters define how the model should send its messages.
-   Press "SET PUBLICATION" to configure the publish parameters.
-#. Set the Publish Address to the first unicast address of the Mesh Light node.
-   Set the Retransmit Count to zero ("Disabled") to prevent the model from sending each button press multiple times.
-   The rest of the publish parameters can be left at their default values.
-   Press "APPLY" to accept the configuration.
-#. You should now be able to control the first LED on the Mesh Light device by pressing Button 1 on the Mesh Light Switch device kit.
-
-Repeat steps 7-10 for each of the elements on the node to control each of the four LEDs on the Mesh Light device.
+#. Tap :guilabel:`Generic OnOff Client` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   1. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+#. Configure the Client model publish parameters, which define how the model will send its messages:
+
+   1. Tap :guilabel:`SET PUBLICATION`.
+   #. Set the Publish Address to the first unicast address of the Mesh Light node.
+   #. Set the Retransmit Count to zero (:guilabel:`Disabled`) to prevent the model from sending each button press multiple times.
+   #. Leave the rest of the publish parameters at their default values.
+   #. Tap :guilabel:`APPLY` to confirm the configuration.
+
+#. Tap :guilabel:`ON` to light up the first LED on the development kit.
```

Comment:
This step shouldn't be part of the light switch guide. It does not work for this sample, it's only for the mesh light.